designer/client/src/components/graph/node-modal/fragment-input-definition/settings/variants/StringBooleanVariants/AnyValueVariant.tsx

```diff
@@ -1,27 +1,36 @@
 import React from "react";
-import { ValidationsFields } from "../fields/ValidationsFields";
 import InitialValue from "../fields/InitialValue";
 import { SettingLabelStyled, SettingRow } from "../fields/StyledSettingsComponnets";
 import { TextAreaNodeWithFocus } from "../../../../../../withFocus";
 import { AnyValueParameterVariant, onChangeType } from "../../../item";
 import { VariableTypes } from "../../../../../../../types";
 import { useTranslation } from "react-i18next";
+import { Error } from "../../../../editors/Validators";
 
 interface Props {
     item: AnyValueParameterVariant;
     onChange: (path: string, value: onChangeType) => void;
     path: string;
     variableTypes: VariableTypes;
     readOnly: boolean;
+    fieldsErrors: Error[];
 }
 
-export const AnyValueVariant = ({ item, path, onChange, readOnly, variableTypes }: Props) => {
+export const AnyValueVariant = ({ item, path, onChange, readOnly, variableTypes, fieldsErrors }: Props) => {
     const { t } = useTranslation();
 
     return (
         <>
             {/*<ValidationsFields path={path} item={item} onChange={onChange} variableTypes={variableTypes} />*/}
-            <InitialValue path={path} item={item} onChange={onChange} readOnly={readOnly} variableTypes={variableTypes} />
+            <InitialValue
```

Comment:
The type of initial value is not validated right now in case of AnyValue setting. Is this BE's job?